README.md

```diff
@@ -1 +1,412 @@
-# Writing Initializers
\ No newline at end of file
+# Writing Initializers
+
+## Objectives
+
+1. Recognize when initializers are useful, especially with read-only properties.
+2. Identify the role of a designated initializer.
+3. Compose a designated initializer.
+4. Override the default initializer (`init`) to call a designated initializer.
+5. Compose convenience initializers that call a designated initializer.
+
+## Initializer Methods
+
+We've discussed the use cases of the `alloc` and `init` method pairs in the past, but what are these methods actually doing? The `alloc` method is a class method that *allocates* a place in memory (RAM) to hold the instance variables that comprise an object of the given class. The `init` method is what actually populates that portion of memory with the relevant structure. Other initializers can be written which actually populate the structure with information (values) right from the start.
+
+We can compare these two roles to the phases of constructing a building: the `alloc` method is like selecting the ground that will be the building's lot, designating and preparing the place for it to go. The `init` method, however, is the phase of actually building the structure that's detailed in the architect's blueprint.
+
+The `init` method is actually just the basic initializer, called the "default initializer", but an initializer is any method that returns an `instancetype` and the compiler will require that the method name begins with `init...`. Methods from the `init` family can be set up to offer varying levels of pre-fabricated setup. 
```

Comment:
That first sentence reads weird. Maybe:
"The `init` method is actually just the basic initializer, called the "default initializer". In general, though, an initializer is any method that returns `instancetype` and begins with `init...`.